androbot/config.py

```diff
@@ -8,9 +8,13 @@ class Settings(BaseSettings):
     db_username: str = Field(..., env="DB_USERNAME")
     db_password: str = Field(..., env="DB_PASSWORD")
     db_host: str = Field(..., env="DB_HOST")
-    db_port: str = Field(..., env="DB_PORT")
+    db_port: int = Field(5432, env="DB_PORT")
     db_name: str = Field(..., env="DB_NAME")
-    static_folder: Path = Field(..., env="STATIC_FOLDER")
+    static_folder: Path = Field("templates", env="STATIC_FOLDER")
+    fsm_redis_host: str = Field("", env="REDIS_HOST")
```

Comment:
Лучше все же так imo

```suggestion
    fsm_redis_host: Optional[str] = Field(None, env="REDIS_HOST")
```